mock/cpp_mock_scenarios/src/lane_change/lanechange_left.cpp

```diff
@@ -30,9 +29,7 @@ class LaneChangeLeftScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit LaneChangeLeftScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "lanechange_left", ament_index_cpp::get_package_share_directory("kashiwanoha_map") + "/map",
```

Comment:
.osm file should be exactly fixed in scenario.
So, please specify it in constructor.